extension/mooltipass-content.js

```diff
@@ -1453,6 +1453,7 @@ cip.doSubmit = function doSubmit(pass)
 }
 
 cip.retrieveCredentialsCallback = function (credentials, dontAutoFillIn) {
+	if (!credentials) credentials = [];
```

Comment:
Not a bug per se, as credentials should always come with something, but it if comes with null/undefined it fails in the client side (at line 1458/1459 it asumes it is a proper Array) and from there, the extension stops working and you need to reload it.